package.json

```diff
@@ -23,10 +23,12 @@
     "eslint-config-airbnb-base": "11.3.1",
     "eslint-plugin-import": "2.7.0",
     "express": "4.15.3",
+    "grpc": "^1.4.1",
```

Comment:
@Dave-Gallup please remove the `^` from this dependency we futzed with yesterday so as to lock in its version, thanks!